### ISSUE TITLE
Wrong import in class Document?

### ISSUE THREAD

**Issue by jowerner** [HtmlUnit/htmlunit#441]
Class `Document` makes use of `HTMLFieldSetElement` now. To this end, it imports `org.w3c.dom.html.HTMLFieldSetElement`. Is this intended or should this rather be `com.gargoylesoftware.htmlunit.javascript.host.html.HTMLFieldSetElement`? Thanks!

**Comment by rbri** [comment]
LoL - it should be a Frameset - will fix that soon (and write the last missing test).
Many thanks - as always

**Comment by jowerner** [comment]
Oh ... :laughing:

**Comment by rbri** [comment]
Fixed, but how did you found that? Have you done some architecture tests?

**Comment by jowerner** [comment]
No, I just happened to compile HU with a JDK 11. In Java 11, when the compiler sees that a certain package is referenced, it checks if that very package is provided by exactly one module only and complains otherwise.

HU and its dependencies have an overlap with standard Java 11 modules causing such compiler errors. When excluding `xml-apis.jar` from HU and the `jdk.jsobject` module from Java 11, there was only one compile error left, in class `Document`. When I tried to resolve that error as well, I noticed that the one import statement looked suspicious.
